pkg/vault/contracts/CompositeLiquidityRouter.sol

```diff
@@ -236,8 +261,9 @@ contract CompositeLiquidityRouter is ICompositeLiquidityRouter, BatchRouterCommo
     }
 
     function addLiquidityERC4626PoolProportionalHook(
-        AddLiquidityHookParams calldata params
-    ) external nonReentrant onlyVault returns (uint256[] memory underlyingAmountsIn) {
+        AddLiquidityHookParams calldata params,
+        bool[] calldata useWrappedTokens
+    ) external nonReentrant onlyVault returns (uint256[] memory amountsIn) {
         IERC20[] memory erc4626PoolTokens = _vault.getPoolTokens(params.pool);
         uint256 poolTokensLength = erc4626PoolTokens.length;
 
```

Comment:
// Revert if `tokensIn` length does not match `maxAmountsIn` and `useWrappedTokens`.
        InputHelpers.ensureInputLengthMatch(poolTokensLength, params.maxAmountsIn.length, useWrappedTokens.length);